### PR TITLE
Add Torchserve to trusted-deep-learning.

### PR DESCRIPTION
**Pull request by Hanyu-Jin**
## Description

Add Torchserve to trusted-deep-learning.

### 1. Why the change?

Enable Torchserve in trusted-deep-learning.

### 2. User API changes

You can use Torchserve in sgx like following:
bash start-torchserve-sgx.sh -c your_config_file

### CODE REVIEW

File: ppml/trusted-deep-learning/base/Dockerfile [hunk 1]
```diff
@@ -61,7 +89,23 @@ RUN env DEBIAN_FRONTEND=noninteractive apt-get update && \
...
+    patch /ppml/bash.manifest.template /bash.manifest.template.patch && \
+# Torchserve
+    pip install --no-cache-dir torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cpu && \
+    pip install --no-cache-dir cython pillow==9.0.1 captum packaging nvgpu && \
+    pip install --no-cache-dir torchserve==0.6.1 torch-model-archiver==0.6.1 torch-workflow-archiver==0.2.5 && \
+    apt-get install -y openjdk-${JDK_VERSION}-jdk && \
+    mkdir -p ${JAVA_HOME} && \
+    cp -r /usr/lib/jvm/java-${JDK_VERSION}-openjdk-amd64/* ${JAVA_HOME} && \
+    sed -i '/MAX_FAILURE_THRESHOLD = 5/ios.environ\[\"MPLCONFIGDIR\"\]=\"\/tmp\/matplotlib\"' /usr/local/lib/python3.7/dist-packages/ts/model_service_worker.py && \
+    sed -i '/import abc/iimport sys' /usr/local/lib/python3.7/dist-packages/ts/torch_handler/base_handler.py && \
```
Comment (gc-fu) [inline ppml/trusted-deep-learning/base/Dockerfile:101]: Is iimport sys a typo or part of the grammar used in `sed` command?

File: ppml/trusted-deep-learning/base/Dockerfile [hunk 2]
```diff
@@ -26,6 +47,13 @@ ADD ./pert_ipex.py   /ppml/examples/pert_ipex.py
 ADD ./filelock.patch /filelock.patch
 ADD ./bash.manifest.template.patch /bash.manifest.template.patch
 
+# COPY frontend.jar
+COPY --from=temp /ppml/torchserve/frontend.jar /ppml/torchserve/frontend.jar
+# Examples for Torchserve
+ADD ./start-backend-sgx.sh      /ppml/examples/start-backend-sgx.sh
```
Comment (gc-fu) [inline ppml/trusted-deep-learning/base/Dockerfile]: Consider putting these scripts in separate folders

**Review by gc-fu — APPROVED** [review]
Since you have already tested it, LGTM

**Review by glorysdj — APPROVED** [review]
LGTM

File: ppml/trusted-deep-learning/base/Dockerfile (hunk 2 above)
Comment (Hanyu-Jin) [reply ppml/trusted-deep-learning/base/Dockerfile]: Done.

**Pull request merged**